pkg/vulnsrc/glad/glad_test.go

```diff
@@ -0,0 +1,74 @@
+package glad_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/aquasecurity/trivy-db/pkg/db"
+	"github.com/aquasecurity/trivy-db/pkg/dbtest"
+	"github.com/aquasecurity/trivy-db/pkg/vulnsrc/glad"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestVulnSrc_Update(t *testing.T) {
+	type want struct {
+		key   []string
+		value string
+	}
+	tests := []struct {
+		name       string
+		dir        string
+		wantValues []want
+		wantErr    string
+	}{
+		{
+			name: "happy path go",
+			dir:  filepath.Join("testdata", "happy"),
+			wantValues: []want{
```

Comment:
We should test the `vulnerability-test` bucket as well.

Reply:
Is it different from vulnerability-detail?